content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -1281,7 +1281,28 @@ We have plenty of work in progress! Some of this will likely land in **Bevy 0.14
 
 Check out the [**Bevy 0.14 Milestone**](https://github.com/bevyengine/bevy/milestone/20) for an up-to-date list of current work that contributors are focusing on for **Bevy 0.14**.
 
-* **More editor experimentation:** TODO
+* **More editor experimentation:** Led by the brilliant JMS55, we've opened up a free-form [playground] to define and answer [key questions] about the design of the `bevy_editor`: not through discussion, but through concrete prototyping.
+Should we use an in-process editor (less robust to game crashes) or an external one (more complex)?
+Should we ship an editor binary (great for non-programmers) or embed it in the game itself (very hackable)?
+Let's find out by doing!
+
+There are some incredible mockups, functional prototypes and third-party editor-ajdacent projects out there. Some highlights:
+
+* [] a Bevy-branded editor UI mockup by `@Jacob_` and `@!!&Amy` on Discord, imagining what the UX for an ECS-based editor could look like
+* [] [`bevy_animation_graph`]: a fully-functional asset-driven animation graph crate with its own node-based editor for Bevy
+* [] [`space_editor`]: a polished Bevy-native third-party scene editor that you can use today!
+* [] [`Blender_bevy_components_workflow`]: an impressively functional ecosystem of tools that lets you use Blender as a seamless level and scene editor for your games today.
+* [] `@coreh`'s experiment with reflection-powered inter-process communication, allowing devs to inspect and control their Bevy games from other processes, languages and even devices! [Try it out live]!
```

Comment:
```suggestion
* [] `@coreh`'s experiment on a [reflection-powered remote protocol], allowing devs to inspect and control their Bevy games from other processes, languages and even devices! [Try it out live]!
```

To avoid sounding redundant with the mention of processes later in the same line, and also mentioning the idea of making into a well defined a protocol.